### PR TITLE
MX queue collect input blocking

### PR DESCRIPTION
**Pull request by mskinner5278**
mx subpatch 2
* Queue Collect checkbox now switches selected sample instead of blocking user input

### CODE REVIEW

File: lsdcGui.py [hunk 1]
```diff
@@ -4391,9 +4391,8 @@ def addRequestsToAllSelectedCB(self):
           self.selectedSampleID = itemData
           if (getBlConfig("queueCollect") == 0):
             if (self.mountedPin_pv.get() != self.selectedSampleID):                    
+              self.selectedSampleID = self.mountedPin_pv.get()
```
Comment (JunAishima) [inline lsdcGui.py:4394]: is this meant to always queue a collection (because the return is removed), but always leave the sample selected?

Comment (mskinner5278) [reply lsdcGui.py:4394]: Correct, they felt it was unnecessarily stopping operations every time it happened, so the return was removed.

Comment (JunAishima) [reply lsdcGui.py:4394]: ok, looks strange, but the beamlines asked for it so I'm ok with it.

File: lsdcGui.py [hunk 2]
```diff
@@ -4427,8 +4426,8 @@ def addSampleRequestCB(self,rasterDef=None,selectedSampleID=None):
 
       if (getBlConfig("queueCollect") == 0):
         if (self.mountedPin_pv.get() != self.selectedSampleID):                    
+          self.selectedSampleID = self.mountedPin_pv.get()
           self.popupServerMessage("You can only add requests to a mounted sample, for now.")
```
Comment (JunAishima) [inline lsdcGui.py:4430]: Did Edwin want these popup messages to be removed? It seems like the behavior is different now, so it should either not be displayed at all or show something different.

**Pull request merged**